### PR TITLE
Sample: Showcase 2:1 aspect ratio

### PR DESCRIPTION
**Pull request by vanniktech**
Also the previous code did things different than the text shown in the UI

### CODE REVIEW

**Comment by Canato** [comment]
We can add a new ratio, but `4:3` differ from `16:9`
ratio

File: sample/src/main/java/com/canhub/cropper/sample/options_dialog/SampleOptionsBottomSheet.kt [hunk 1]
```diff
@@ -96,7 +96,7 @@ internal class SampleOptionsBottomSheet : BottomSheetDialogFragment() {
 
         when (options.ratio) {
             Pair(1, 1) -> binding.ratio.chipOneOne.isChecked = true
-            Pair(4, 3) -> binding.ratio.chipFourThree.isChecked = true
+            Pair(2, 1) -> binding.ratio.chipTwoOne.isChecked = true
```
Comment (Canato) [inline sample/src/main/java/com/canhub/cropper/sample/options_dialog/SampleOptionsBottomSheet.kt:99]: We can add a new one, but let's not remove `4:3`

**Review by Canato — CHANGES_REQUESTED** [review]

**Comment by Canato** [comment]
On another note, we need to update the CHANGELOG

**Comment by vanniktech** [comment]
> On another note, we need to update the CHANGELOG

With what?

**Comment by Canato** [comment]
> > On another note, we need to update the CHANGELOG
> 
> With what?

Add ratio `2,1`

**Comment by vanniktech** [comment]
Will do once the other PR is merged, otherwise I get conflicts in the changelog file

**Review by Canato — APPROVED** [review]

**Review by Canato — CHANGES_REQUESTED** [review]
Some merge conflicts. Request changes so you get notify

**Review by Canato — APPROVED** [review]

**Pull request merged**